CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Scout Changelog
```

Comment:
A little confused about what's going to happen here. Since we've edited the changelog too...